addons/project/models/project.py

```diff
@@ -1079,48 +1079,48 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     ], default='this', store=False)
     recurrence_message = fields.Char(string='Next Recurrencies', compute='_compute_recurrence_message')
```

Comment:
I think we could also add the group for this field since the compute will read some fields in the `project.task.recurrence` model.

Reply:
@xavierbol Sorry for the late response, thanks for your suggestion. I fixed it according to your suggestion

Reply:
No problem 🙂 

I am wondering if we should not also add the group on `recurrence_message` field. What do you think? 🙂

Reply:
@xavierbol okey, done